docs/Makefile

```
@@ -3,7 +3,7 @@
 
 # You can set these variables from the command line.
 SPHINXOPTS    =
-SPHINXBUILD   = sphinx-build
+SPHINXBUILD   = ../venv/bin/sphinx-build
```

I'll revert this since it's very specific to use of venv
```suggestion
SPHINXBUILD   = sphinx-build
```